src/components/StartingAddressIndexPicker.jsx

```diff
@@ -19,18 +19,19 @@ import {
 import { wrappedActions } from "../actions/utils";
 import { SET_STARTING_ADDRESS_INDEX } from "../actions/settingsActions";
 
-class StartingAddressIndexPicker extends React.Component {
-  constructor(props) {
-    super(props);
+const StartingAddressIndexPicker = ({
+  startingAddressIndex,
+  setStartingAddressIndex,
+}) => {
+  // Ask @winsby about these state initiations...
+  const [customIndex, setCustomIndex] = useState(startingAddressIndex !== 0);
+  const [startingAddressIndexField, setStartingAddressIndexField] =
+    useState(startingAddressIndex);
+  // What is the correct state initiation here? Didn't see it in the OG file
+  const [startingAddressIndexError, setStartingAddressIndexError] =
+    useState("");
```

Comment:
These should be good.
```suggestion
  const [customIndex, setCustomIndex] = useState(startingAddressIndex !== 0);
  const [startingAddressIndexField, setStartingAddressIndexField] =
    useState(startingAddressIndex);
  const [startingAddressIndexError, setStartingAddressIndexError] =
    useState("");
```